src/app/shared/_themes.scss

```diff
@@ -240,6 +246,7 @@ $theme-night-border: 2px solid #00c0ff;
   $theme-night-wrapper-mobile-background-color,
   $theme-night-text-color,
   $theme-night-text-color,
+  $theme-night-subtext-color,
```

Comment:
We'll have to add it for the black theme as well, or the arguments may be out of place